Gruntfile.js

```diff
@@ -7,10 +7,11 @@ module.exports = function(grunt) {
 	grunt.loadNpmTasks('grunt-contrib-clean');
 
 	var DIST = 'dist/',
-		DIST_OPTIMIZED = DIST + 'optimized/',
-		DIST_ANDROID = DIST + 'android/',
+		DIST_OPTIMIZED = DIST + 'svg/optimized/',
 		DIST_IOS = DIST + 'iOS/',
-		DIST_SPRITE = DIST + 'sprite/',
+		DIST_SPRITE = DIST + 'svg/sprite/',
```

Comment:
I take that back, I'm favoring bumping the major version instead. `optimized` also changes here. The restructuring seems useful for future formats (android)